src/index.js

```diff
@@ -10,6 +10,11 @@ import { createClient } from '~/apollo/createClient'
 import './index.scss'
 
 require('./ethers.extension')
+require('./intercom')
+
+window.Intercom('boot', {
```

Comment:
Should this be conditional based on whether REACT_APP_INTERCOM_APP_ID exists?

Reply:
Yes. Definitely